web/src/main/java/se/inera/intyg/webcert/web/service/arende/ArendeServiceImpl.java

```diff
@@ -216,6 +220,9 @@ public Arende processIncomingMessage(Arende arende) {
             monitoringLog.logArendeReceived(certificateId, utkast.getIntygsTyp(), utkast.getEnhetsId(), arende.getAmne(),
                 arende.getKomplettering().stream().map(MedicinsktArende::getFrageId).collect(Collectors.toList()),
                 arende.getSvarPaId() != null);
+            Arende saved = arendeRepository.save(arende);
+            sendNotificationAndCreateEventForIncomingMessage(saved, utkast.getVardgivarId(), utkast.getSignatur().getSigneringsDatum());
+            return saved;
```

Comment:
If you like you can remove the "else" as the if contains a return. This will simplify the code a little bit.